layouts/fastly/helix.vcl

```diff
@@ -1781,23 +1785,25 @@ sub hlx_bereq {
   if (req.http.X-Request-Type != "Preflight") {
     unset bereq.http.X-Orig-Url;
     unset bereq.http.X-Orig-Host;
+    unset bereq.http.X-Owner;
+    unset bereq.http.X-Repo;
+    unset bereq.http.X-Ref;
+    # unset bereq.http.X-Repo-Root-Path;   // todo: should we remove this as well ?
   }
   unset bereq.http.X-Backend-URL;
   unset bereq.http.X-Request-Type;
   unset bereq.http.X-Static-Content-Type;
   unset bereq.http.X-Allow;
   unset bereq.http.X-Deny;
-  unset bereq.http.X-Owner;
   unset bereq.http.X-Index;
-  unset bereq.http.X-Repo;
-  unset bereq.http.X-Ref;
   unset bereq.http.X-Root-Path;
   unset bereq.http.X-Action-Root;
   unset bereq.http.X-Github-Static-Repo;
   unset bereq.http.X-Github-Static-Owner;
   unset bereq.http.X-Github-Static-Root;
   unset bereq.http.X-Github-Static-Ref;
   unset bereq.http.X-Restarts;
+  # unset bereq.http.X-Trace; // todo: should we remover this as well ?
```

Comment:
@trieloff this was not removed from the bereq before... is it important to the backends ?

Reply:
```suggestion
  unset bereq.http.X-Trace; // todo: should we remover this as well ?
```

Good idea to remove this.